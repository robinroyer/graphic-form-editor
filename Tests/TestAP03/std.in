PL p01 10 10
PL p01 42 42
EXIT