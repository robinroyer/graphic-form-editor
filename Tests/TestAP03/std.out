OK
#New object: p01